scripts/tests/test_depot_authority_sentinel.py

```
@@ -0,0 +1,460 @@
+            "pr_windows.yml",
+        }
+        validation_entrypoints = {
+            path.name
+            for path in WORKFLOWS.glob("pr_*.yml")
+            if "  pull_request:" in path.read_text(encoding="utf-8")
+        }
+        self.assertEqual(validation_entrypoints, expected)
+
+        for name in ("ci-quality-lane.yml", *sorted(expected)):
+            current = (WORKFLOWS / name).read_text(encoding="utf-8")
+            baseline_entry = subprocess.check_output(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)